Makefile

```diff
@@ -6,57 +6,89 @@ EXTERNAL_TOOLS=\
 	github.com/kardianos/govendor
 BUILD_TAGS?=${TOOL}
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
-# bin generates the releaseable binaries for this plugin
+
+PLUGIN_NAME?=$(shell command ls bin/)
```

Comment:
The plugin does not use the cmd folder pattern. I used the bin output as a fallback as it seemed logical since the make target that uses this variable builds the project before accessing it, but yeah it could create weird error messages.

Dunno if there is a more sensible way to derive the plugin name. We could also just hardcode the string, the value is not that important even "potato" works as it always use the same reference during the config steps.

Reply:
Ah, missed that detail! I'm good with how it is using bin now as long as things work from a fresh clone 👍